sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTraceExporter.cs

```diff
@@ -38,7 +38,7 @@ public override ExportResult Export(in Batch<Activity> batch)
             }
             catch (Exception ex)
             {
-                AzureMonitorTraceExporterEventSource.Log.FailedExport(ex);
+                AzureMonitorTraceExporterEventSource.Log.WriteException(new Exception("Failed to export activities", ex));
```

Comment:
This could be heavy - even if the log is turned off, we still pay for the exception creation.

Reply:
I agree. this was a mistake on my part.